config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "process.env.DB_USER",
+    "password": "process.env.DB_PASSWORD",
+    "database": "process.env.DB_DATABASE",
+    "host": "process.env.DB_HOST",
```

Comment:
The value for `host` should be accessed using `process.env.DB_HOST` without quotes to correctly retrieve the environment variable.